client/stylesheets/_responsive.scss

```diff
@@ -85,24 +79,37 @@
 }
 
 @media (max-width: 900px) {
-	.helpThree {
-		width: 80%;
-		margin: 0 auto 40px auto;
-	}
+    .helpThree {
+        width: 80%;
+        margin: 0 auto 40px auto;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 auto 40px` instead of `0 auto 40px auto`